mlos_bench/mlos_bench/tests/optimizers/conftest.py

```diff
@@ -24,6 +41,7 @@ def mock_opt(tunable_groups: TunableGroups) -> MockOptimizer:
         config={
             "minimize": "score",
             "max_iterations": 5,
+            "start_with_defaults": True,
```

Comment:
Necessary?  Shouldn't that be the default?

Reply:
made `True` teh default